### PR TITLE
DFPL-1037: Add 'Manage Documents' To Cafcass Login Permissions

### PR DESCRIPTION
**Pull request by qasimnawaz-moj**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DFPL-1037

### Change description ###

Add 'Manage Documents' To Cafcass Login Permissions

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by AlistairEdwinOsborne — APPROVED** [review]
all good!

**Review by AlistairEdwinOsborne — CHANGES_REQUESTED** [review]
In response to Dan's comment, might be a good idea to double check!

**Review by DanCatchpole — CHANGES_REQUESTED** [review]
Think caseworker-publiclaw-cafcass needs to be added to AuthorisationCaseEvent too

File: ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -649,7 +649,8 @@
           "caseworker-publiclaw-judiciary",
           "caseworker-publiclaw-magistrate",
           "caseworker-publiclaw-courtadmin",
-          "caseworker-publiclaw-gatekeeper"
```
Comment (DanCatchpole) [inline ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent.json:652]: think they only need access to this event (the others are specifically for LAs + solicitors to be restricted to certain types of docs I believe

**Review by DanCatchpole — APPROVED** [review]
think it all looks good! I imagine if there's a missing field anywhere the QAs will spot that quicker than I can 🤣

**Review by DanCatchpole — DISMISSED** [review]

**Pull request merged**